src/site/markdown/server/manage-webadmin.md

```diff
@@ -591,6 +592,329 @@ Response codes:
  - 400: Group structure or member is not valid
  - 500: Internal error
 
+## Administrating mail repositories
+
+### Listing mail repositories
+
+```
+curl -XGET http://ip:port/mailRepositories
+```
+
+The answer looks like:
+
+```
+[
+    {
+        "repository": "file://var/mail/error/",
+        "encodedUrl": "file%3A%2F%2Fvar%2Fmail%2Ferror%2F"
+    },
+    {
+        "repository": "file://var/mail/relay-denied/",
+        "encodedUrl": "file%3A%2F%2Fvar%2Fmail%2Frelay-denied%2F"
+    },
+    {
+        "repository": "file://var/mail/spam/",
+        "encodedUrl": "file%3A%2F%2Fvar%2Fmail%2Fspam%2F"
+    },
+    {
+        "repository": "file://var/mail/address-error/",
+        "encodedUrl": "file%3A%2F%2Fvar%2Fmail%2Faddress-error%2F"
+    }
+]
+```
+
+You can use `encodedUrl` to access the repository.
+
+Response codes:
+
+ - 200: The list of mail repositories
+ - 500: Internal error
+
+### Getting additional information for a mail repository
+
+```
+curl -XGET http://ip:port/mailRepositories/encodedUrlOfTheRepository/
+```
+
+Resource name `encodedUrlOfTheRepository` should be the encoded URL of an existing mail repository. Example:
```

Comment:
(given by the id field)

Reply:
it's not the resource name but it's id then